src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--30">
+      <div class="calendar_block calendar_block--start-day--sat"></div>
```

Comment:
`start-day` modifier should be assigned to the `calendar` block, not to the `calendar_block` element